server/services/database/profiles.js

```diff
@@ -21,7 +22,18 @@ const createProfile = (newProfile) => {
 				newProfile.profile_type,
 			],
 		)
-		.then((result) => result.rows);
+		.then((result)=>{
+			let profileId=result.rows[0].id;
+			let values=[];
+			newProfile.groups.forEach((groupId) => {
+				values.push([profileId,groupId]);
+			});
```

Comment:
```suggestion
			const values = newProfile.groups.map(groupId => [profileId, groupId]);
```
We don't need to change the code here, but I just wanted to point out that an alternative solution is that we can create this `values` array using `map` since we are transforming an array of groups to another array of the same length 😃